client-misk-embedded/gradle.properties

```diff
@@ -0,0 +1,4 @@
+POM_ARTIFACT_ID=client-misk
```

Comment:
should be -embedded

Reply:
Yeah I thought I had deleted this... It is gone now.